src/lib.rs

```diff
@@ -256,7 +265,7 @@ mod aes128gcm_tests {
         .unwrap_err();
         match err.kind() {
             ErrorKind::HeaderTooShort => {}
-            _ => assert!(false),
+            _ => panic!("Unexpected error type!"),
```

Comment:
Nit: `unreachable!` probably is more appropriate, although there's no difference in practice.